src/site/components/events_list.drupal.liquid

```diff
@@ -0,0 +1,30 @@
+{% for event in reverseFieldOfficeNode.entities%}
+<div class="vads-u-margin-bottom--2">
+  <h4><a href="{{event.entityUrl.path}}">{{event.title}}</a></h4>
+  <p>{{event.fieldDescription}}</p>
+  <div class="usa-grid usa-grid-full">
```

Comment:
This is more of a suggestion than a thing you have to change, but using the responsive grid (https://design.va.gov/layout/flexbox-grid) like you did on the other pages is preferred to the `usa-grid` grid.

Reply:
I was able to replace in a couple of places, but backed off elsewhere (things started blowing up), @jbalboni